### PR TITLE
asserts: introduce assertstest with helpers to test code involving assertions

### PR DESCRIPTION
**Pull request by pedronis**

### CODE REVIEW

File: asserts/assertstest/assertstest.go [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+	pkt, err := packet.Read(body)
+	if err != nil {
+		panic(err)
+	}
+
+	pkPkt := pkt.(*packet.PrivateKey)
+	return asserts.OpenPGPPrivateKey(pkPkt), pkPkt
+}
+
+// A sample developer key.
```
Comment (emgee) [inline asserts/assertstest/assertstest.go]: Why *developer* key? Surely this could be used for other reasons too, e.g. as a root key.

Comment (emgee) [inline asserts/assertstest/assertstest.go]: @pedronis mentioned we also have systestkeys, so we probably have pre-generated keys for most things now.

Maybe it's worth mentioning systestkeys in this key's comment?

**Comment by emgee** [comment]
+1

File: asserts/assertstest/assertstest.go (hunk 1 above)
Comment (pedronis) [inline asserts/assertstest/assertstest.go]: @emgee done

**Pull request merged**